src/Graphics/OpenGLContext/GLSL/glsl_CombinerProgramBuilder.cpp

```diff
@@ -1117,27 +1093,29 @@ class ShaderFragmentGlobalVariablesNotex : public ShaderPart
 					"layout(location = 0, index = 0) OUT lowp vec4 fragColor; 	\n"
 					"layout(location = 0, index = 1) OUT lowp vec4 fragColor1;	\n"
 					"#define LAST_FRAG_COLOR vec4(0.0) \n"
-					"#define LAST_FRAG_COLOR_ALPHA 1.0 \n"
+					"#define LAST_FRAG_COLOR_ALPHA 0.0 \n"
 			;
 					;
 		} else if (_glinfo.ext_fetch) {
 			m_part +=
-					"layout(location = 0) inout lowp vec4 realFragColor;				\n"
-					"lowp vec4 fragColor;	\n"
+					"layout(location = 0) inout lowp vec4 fragColor;				\n"
+					"lowp vec4 fragColor1;	\n"
 					"#define LAST_FRAG_COLOR realFragColor \n"
 					"#define LAST_FRAG_COLOR_ALPHA realFragColor.a \n"
 					;
 		} else if (_glinfo.ext_fetch_arm) {
 			m_part +=
 					"OUT lowp vec4 fragColor;	\n"
+					"lowp vec4 fragColor1;	\n"
 					"#define LAST_FRAG_COLOR gl_LastFragColorARM \n"
 					"#define LAST_FRAG_COLOR_ALPHA gl_LastFragColorARM.a \n"
 					;
 		} else {
 			m_part +=
 					"OUT lowp vec4 fragColor;	\n"
+					"lowp vec4 fragColor1;	\n"
 					"#define LAST_FRAG_COLOR vec4(0.0) \n"
-					"#define LAST_FRAG_COLOR_ALPHA 1.0 \n"
```

Comment:
The reason I defined this as 1.0 was because that was the value that was used when initializing muxB when dual source blending was enabled. Was that wrong before?

Reply:
Yes, I noticed it. When muxB == 1 the blender must use memory alpha as input. Depending on the combine mode, this is impossible to emulate with dual source blending, no matter if it is initialized to 0.0 or 1.0. I am not sure why I put 1.0 before, but I can't see why it should be better.

Reply:
Ok, that answers my question.